include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -47,7 +47,7 @@ using namespace cute;
 template <int Stages, class Schedule, class TileShape_, class ElementA_, class StrideA_, class ElementB_, class StrideB_,
           class TiledMma_, class GmemTiledCopyA_, class SmemLayoutAtomA_, class SmemCopyAtomA_, class TransformA_,
           class GmemTiledCopyB_, class SmemLayoutAtomB_, class SmemCopyAtomB_, class TransformB_>
-struct CollectiveMma<MainloopIntelXeXMX16<Stages, Schedule>, TileShape_, ElementA_, StrideA_, ElementB_, StrideB_, TiledMma_,
+struct CollectiveMmaNew<MainloopIntelXeXMX16<Stages, Schedule>, TileShape_, ElementA_, StrideA_, ElementB_, StrideB_, TiledMma_,
```

Comment:
This class needs to stay as a `CollectiveMma`. Instead of changing the name, you can introduce a new dispatch policy to replace `MainloopIntelXeXMX16`. I'd suggest something like `MainloopXeL1Staged` that also better describes what this mainloop does.